lib/reek/report/formatter.rb

```diff
@@ -42,6 +43,11 @@ def format_hash(warning)
         warning.yaml_hash
       end
 
+      # :reek:UtilityFunction
+      def format_code_climate_hash(warning)
+        CodeClimateFormatter.new(warning).to_hash
+      end
```

Comment:
Yeah, I considered that, but thought it might be better to keep it consistent with the existing JSON formatter. WDYT @troessner? I do prefer it the way @chastell suggests.